conf/modules/post_variant_calling.config

```diff
@@ -16,6 +16,30 @@
 
 process {
 
+    withName: 'GERMLINE_VCFS_NORM'{
+        ext.args = { [
+            '--multiallelics - both', //split multiallelic sites into biallelic records and both SNPs and indels should be merged separately into two records
+            '--rm-dup all'            //output only the first instance of a record which is present multiple times
+        ].join(' ') }
+        ext.when   = { params.concatenate_vcfs }
+        publishDir = [
+            mode: params.publish_dir_mode,
+                path: { "${params.outdir}/variant_calling/concat/${meta.id}/" }
+        ]
+    }
+
+        withName: 'VCFS_NORM'{
+        ext.args = { [
+            '--multiallelics - both', //split multiallelic sites into biallelic records and both SNPs and indels should be merged separately into two records
+            '--rm-dup all'            //output only the first instance of a record which is present multiple times
+        ].join(' ') }
+        ext.when   = { params.normalized_vcfs }
+        publishDir = [
+            mode: params.publish_dir_mode,
+                path: { "${params.outdir}/variant_calling/normalized/${meta.id}/" }
+        ]
+    }
+
```

Comment:
This could be condensed into a single configuration. Why are you publishing the normalised vcfs into two different subdirectories?

Reply:
The Tabix below is not published in the same way, we would end up with the tbi in a different directory.

Reply:
I explained it below, two different processes for either normalisation and concatenation of germline vcfs or normalisation of all vcfs.